lib/EphemeralSocket.js

```diff
@@ -175,11 +175,12 @@ EphemeralSocket.prototype._send = function _send(data) {
         that._socketUsed = true;
 
         // Create message
-        var message = new Buffer(data);
+        var message = Buffer.from(data);
+        var msg = message.toString();
 
-        debug(message.toString());
+        debug(msg);
 
-        that._socket.send(message, 0, message.length, that._port, that._hostname);
+        that._socket.send(msg, 0, msg.length, that._port, that._hostname);
```

Comment:
Why include this unrelated change? It introduces a bug for non-ascii messages where the string length isn't equal to the (serialized) buffer length.

Reply:
can you update the branch? thanks.

Reply:
If you push a fix to the branch in your fork, this PR will automatically get updated 🤗

Reply:
updated.